apps/dash-financial-report/app.py

```diff
@@ -65,9 +48,7 @@ def display_page(pathname):
         return overview.create_layout(app)
 
 
-# detail the way that external_css and external_js work and link to alternative method locally hosted
 external_css = [
-    "https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.min.css",
     "https://cdnjs.cloudflare.com/ajax/libs/skeleton/2.0.4/skeleton.min.css",
```

Comment:
Please get rid of these external stylesheets if you've added them into the `assets` folder!

Reply:
Oops!